src/strands/multiagent/a2a/server.py

```diff
@@ -98,8 +98,14 @@ def agent_skills(self) -> list[AgentSkill]:
         Returns:
             list[AgentSkill]: A list of skills this agent provides.
         """
-        # TODO: translate Strands tools (native & MCP) to skills
-        return []
+        skills: list[AgentSkill] = []
+        tool_configs = self.strands_agent.tool_registry.get_all_tools_config().values()
```

Comment:
I presume we don't care if tools are added after the fact?  (I don't have an opinion but strands does support it)

If we wanted to support that, maybe this should be a getter?